reana_workflow_engine_snakemake/config.py

```diff
@@ -17,3 +17,6 @@
 
 DEFAULT_SNAKEMAKE_REPORT_FILENAME = "report.html"
 """Snakemake report default filename."""
+
+SNAKEMAKE_MAX_PARALLEL_JOBS = 100
+"""Snakemake maximum number of jobs that can run in parallel."""
```

Comment:
Should we create a Helm configuration variable for this, so that sites can amend it more easily to their needs?

Reply:
Not sure.. While it's nice to have as much fine-grained configuration as possible, it might introduce unnecessary complexity for admins as it's something very specific.